### ISSUE TITLE
heroku one click deploy is broken

### ISSUE THREAD

**Issue by liberaltech** [RocketChat/Rocket.Chat#4958]
Your Rocket.Chat version: (make sure you are running the latest)
Is anyone else having this issue?

**Comment by tarikfayad** [comment]
I'm having the same issue. It looks like it's an issue with the `meteor-dev-bundle`:
![screen shot 2016-11-16 at 3 25 43 pm](https://cloud.githubusercontent.com/assets/6421230/20364398/ff74c81e-ac10-11e6-8be8-256782e9f4f6.png)

**Comment by liberaltech** [comment]
@tarikfayad I cloned the Source from https://github.com/RocketChat/Rocket.Chat.git
reinstalled meteor using curl. It gave me one more issue saying my "app is crashing" so I opened the browser using localhost:3000, this game me instructions to install a package but I forget which one. Once I did all f this I eventually got it to run locally. 
I also install bcrypt after it got running to solve a minor error it kept giving me. 
Im still having a hard time deploying it to Heroku.
